appveyor.yml

```diff
@@ -15,9 +15,8 @@ build_script:
   - dotnet build ./NVelocity.sln -c Release 
 
 test_script:
-  - dotnet test ./src/NVelocity.Tests
+  - dotnet test ./src/NVelocity.Tests --no-build -c Release
```

Comment:
So cos the unit tests re-built under 'debug' it caused the NuGet to be created ad debug. So I changed it to not rebuild and use the Release version to run tests against.